canonical-websites/www.ubuntu.com
# Added a print stylesheet

## Done
* Added a new [print stylesheet](http://0.0.0.0:8001/static/css/print.css)


## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- print any page to see that the header, nav, footer and back-to-top links are removed
- see the margins look ok
- see that external links urls are printed
